tests/integration/docker_utils/test_docker.py

```diff
@@ -1186,6 +1186,8 @@ def test_get_container_ip(self, docker_client: ContainerClient, dummy_container)
         assert is_ipv4_address(ip)
         assert "127.0.0.1" != ip
 
+
+class TestDockerImages:
```

Comment:
Started introducing some grouping here, to pull some of the test methods out of the generic `TestDockerClient` class. We can continue with more restructuring as we go.. 👍